README.md

```diff
@@ -37,5 +37,7 @@ Here's the steps to setup the project locally:
 1. `poetry shell`
 2. `poetry install`
 3. `make up`
-4. `make api` or `ENV=local make api`
-5. `ENV=test make tests`
+4. `make ollama model=nomic-embed-text:latest`
```

Comment:
lets make it a specific entry like

### Llama 3.2 with nomic

1. `make ollama model=nomic-embed-text:latest`
2. make ollama model=llama3.2:latest`
3. `make api` or `ENV=local make api`